codebasin.py

```diff
@@ -109,6 +109,15 @@ def guess_project_name(config_path):
         default=False,
         help="Set batch mode (additional output for bulk operation.)",
     )
+    parser.add_argument(
+        "-x",
+        "--exclude",
+        dest="excludes",
+        metavar="<pathspec>",
+        action="append",
+        default=[],
+        help="paths to exclude from the codebase",
```

Comment:
Not sure if this is a good place to have it, but probably needs to have unambiguous spec on how to specify paths:

```console
# this?
--exclude=["*.ext, "*.md"]

# or this?
--exclude="*.ext *.md"
```

Reply:
Maybe to append to your suggestion, "...from the codebase. Multiple excludes can be specified.`?

Reply:
Good idea.  Fixed in 9c896f1348c50ff8f7e8b9403341ffdfd183dc07.

I'm also going to open an issue so we don't lose track of the fact that the help strings all have inconsistent formatting.